app/controllers/webauthn_credentials_controller.rb

```diff
@@ -0,0 +1,22 @@
+class WebauthnCredentialsController < ApplicationController
+  before_action :redirect_to_signin, unless: :signed_in?
+
+  def index
+    @user = current_user
+  end
+
+  def destroy
```

Comment:
Is it acceptable to allow users to delete the key without verifying that they have access to the key?